Framework/Algorithms/src/DiffractionFocussing2.cpp

```diff
@@ -307,9 +307,11 @@ void DiffractionFocussing2::exec() {
                    std::divides<double>());
     // Now multiply by the number of spectra in the group
     std::transform(Yout.begin(), Yout.end(), Yout.begin(),
-                   std::bind2nd(std::multiplies<double>(), groupSize));
+                   std::bind2nd(std::multiplies<double>(),
+                                static_cast<double>(groupSize)));
```

Comment:
Would you replace the deprecated [`std::bind2nd`](http://en.cppreference.com/w/cpp/utility/functional/bind12) with either `std::bind` or a lambda expression?